README.md

```diff
@@ -136,11 +125,13 @@ See https://github.com/vimc/orderly.server/pull/30/commits/29376c630d57c0f74c5d0
 
 # Testing
 
-For local testing, bring up redis docker container via
```

Comment:
redis is needed to run for most of the tests, not just the `test-z-integration.R` tests. Any test which goes through rrq uses it